docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -0,0 +1,105 @@
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| productSet                                            | An arbitrary string used to group products in a set of upgrades & downgrades.                                                                                                                   |
+| productOrder                                          | A number used for sorting products in a set                                                                                                                                                     |
+| webIconURL                                            | Image URL for product icon in web content                                                                                                                                                       |
+| capabilities:{clientID} capabilities:dcdb5ae7add825d2 | Comma-separated list of capabilities enabled by this product for the Relying Party identified by {clientID}                                                                                     |
+| upgradeCTA                                            | HTML content string describing available upgrades from this plan. By convention, should include a link back to a product lead page. That lead page links back to FxA's plan subscription pages. |
+| downloadURL                                           | The download URL for the product.                                                                                                                                                               |
+| emailIconURL                                          | Image URL for product icon in email content                                                                                                                                                     |
+
+##### Plan Metadata
+
+| Key              | Value                                                               |
+| ---------------- | ------------------------------------------------------------------- |
+| previous_plan_id | The value of the previous plan that the user had been subscribed to |
+| plan_change_date | Unix timestamp of the date the plan was change                      |
+
+## Navigating the Payment Flow
+
+Once your API keys are set, restart the affected servers (`auth` or `payments`) if needed.
+
+Reference the [workflow](https://github.com/mozilla/fxa#workflow) section of the FxA docs to sign up for and verify an account. You should now be able to access the payment flow at:
+
+```
+http://127.0.0.1:3031/products/{productId}?plan={planId}
```

Comment:
huh, TIL.  I've been using http://127.0.0.1:3030/subscriptions/products/{productId} for its redriect.

Reply:
This URL only works if you've previously visited a content-server route and picked up an access token in localstorage. It will stop working after that access token expires.

Reply:
FWIW, the redirect also add some flow metric query params.  Not crucial in local dev unless you are working on metrics.

Reply:
Swapping this for the subscriptions one since it handles the redirect.